pipelines/skylab/multiome/Multiome.changelog.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.6
+2024-01-24 (Date of Last Commit)
+
+* Added task GetNumSplits before FastqProcess ATAC task to determine the number of splits based on the bwa-mem2 machine specs
+* Added error message the BWAPairedEndAlignment ATAC task to ensure that the number of splits equals to number of ranks
```

Comment:
```suggestion
* Added an error message to the BWAPairedEndAlignment ATAC task to ensure that the number of splits equals to number of ranks
Comment

Reply:
Two error messages were added. But I can split it and say one was added to compare the number of splits to the number of ranks, and one was added to ensure the same number R1s and R3s are input